src/WSCryptDecryptSoapClient.php

```diff
@@ -104,9 +104,9 @@ private function getStreamContext($testEnv = false, $caFile = null)
             }
         } else {
             $this->streamContextOption['ssl']['peer_name'] = $host;
-            $this->streamContextOption['ssl']['verify_peer_name'] = true;
+            $this->streamContextOption['ssl']['verify_peer_name'] = false;
```

Comment:
also here peer name must be validated to be sure to send money to the real Gestpay server :)